src/MultiOwnable.sol

```diff
@@ -1,124 +1,217 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.4;
 
+/// @notice Storage layout used by this contract.
+///
 /// @custom:storage-location erc7201:coinbase.storage.MultiOwnable
 struct MultiOwnableStorage {
-    /// @dev tracks the index of the next owner added
+    /// @dev Tracks the index of the next owner to add.
     uint256 nextOwnerIndex;
-    /// @dev Allows an owner to be idenfitied by a uint256.
-    /// Some uses--such as signature validation for secp256r1 public
-    /// key owners--requires the caller to assert which owner signed.
-    /// To economize calldata, we allow an index to identify an owner,
-    /// so that the full owner bytes do not need to be passed.
-    /// Note, we use uint256 rather than a smaller uint because it
-    /// provides flexibility at little to no cost.
-    /// uint256 allows that we will (practically) never run out of owner indexes.
-    /// And on L2, where calldata gas is a concern,
-    /// we should not be charged for the extra 0 bytes.
-    mapping(uint256 => bytes) ownerAtIndex;
-    mapping(bytes => bool) isOwner;
+    /// @dev Mapping of indices to raw owner bytes, used to idenfitied owners by their
+    ///      uint256 id (to economize calldata).
+    ///      NOTE: `uint256` rather than a smaller uint because it provides flexibility,
+    ///      in that we will practically never run out of indices.
+    ///
+    ///      In the context of checking whether something was signed by an owner this
```

Comment:
this is not true in the normal ethereum address case because ecrecover returns the address